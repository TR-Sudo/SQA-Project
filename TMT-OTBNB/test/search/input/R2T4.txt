login
user01
search
Oshawa
*
5
logout
quit