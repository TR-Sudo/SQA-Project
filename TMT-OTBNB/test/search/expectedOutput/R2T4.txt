Enter transaction: Enter username: You have been logged in
Enter transaction: Enter city: Enter rental price (0-999.99): Enter number of rooms (0-9): 
Search Results:

Rental ID: a0b0c0d0
City: Oshawa
Number of bedrooms: 5
Rental price per night: 110.0

Rental ID: a0b0c0d2
City: Oshawa
Number of bedrooms: 5
Rental price per night: 110.0

Rental ID: a0b1c3d4
City: Oshawa
Number of bedrooms: 5
Rental price per night: 110.0

Rental ID: a0a0a0a0
City: Oshawa
Number of bedrooms: 5
Rental price per night: 110.0

Rental ID: aaaabbb2
City: Oshawa
Number of bedrooms: 5
Rental price per night: 20.0

End of search results

Enter transaction: You have been logged out
Enter transaction: 